WordPress/Classes/ViewRelated/System/FilterTabBar.swift

```diff
@@ -356,7 +356,19 @@ class FilterTabBar: UIControl {
         }
     }
 
+    var currentlyCelectedItem: FilterTabBarItem? {
+        guard !items.isEmpty, selectedIndex < items.count else {
+            return nil
+        }
+        return items[selectedIndex]
+    }
+
     func setSelectedIndex(_ index: Int, animated: Bool = true) {
+        guard index >= 0, index < items.count else {
```

Comment:
I think you can check if the indices contain this index here:

```suggestion
        guard items.indices.contains(index) else {
```